### PR TITLE
[t428]

### PR DESCRIPTION
**Pull request by secretrobotron**
Added MIT license file and headers to everything I could find.

### CODE REVIEW

File: src/ui/butter.ui.css [hunk 1]
```diff
@@ -1,26 +1,7 @@
...
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-THE SOFTWARE.
-
-**********************************************************************************/
+/* Copyright 2011, 2012 - Mozilla Foundation
```
Comment (jbuck) [inline src/ui/butter.ui.css:1]: Nuke it

File: test/core.js [hunk 2]
```diff
@@ -1,5 +1,10 @@
 /*global text,expect,ok,module,notEqual,test,window*/
 
+/* Copyright 2011, 2012 - Mozilla Foundation
```
Comment (jbuck) [inline test/core.js:3]: Nuke it

**Comment by jbuck** [comment]
Could you add a file (LICENSE_TEMPLATES ?) that contains the two different license header templates that we're using now?

File: test/core.js (hunk 2 above)
Comment (secretrobotron) [inline test/core.js:3]: How'd this escape my search&replace...?

**Pull request merged**